src/Parser/Warlock/Affliction/Modules/Features/ReapBuffTracker.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import Wrapper from 'common/Wrapper';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import { UNSTABLE_AFFLICTION_DEBUFF_IDS } from '../../Constants';
+
+const UA_IDS_SET = new Set(UNSTABLE_AFFLICTION_DEBUFF_IDS);
```

Comment:
Is there any advantage to using a set for this instead of just the array? Otherwise it seems unnecessary. (same in MG and Haunt)